src/component/Button.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const { handleclick } = props;
+  if (props.wide) {
+    return (
+      // eslint-disable-next-line no-useless-concat
+      <button className={'double ' + `${props.color}`} onClick={() => handleclick(props.name)}>{props.name}</button>
```

Comment:
- [ ] no-useless-concat => you could just use es6 entirely. 
```suggestion
      <button className={`double ${props.color}`} onClick={() => handleclick(props.name)}>{props.name}</button>
```